test/clj/game_test/cards/operations.clj

```diff
@@ -1865,6 +1865,69 @@
     (play-from-hand state :corp "Successful Demonstration")
     (is (= 13 (:credit (get-corp))) "Paid 2 to play event; gained 7 credits")))
 
+(deftest surveillance-sweep
+  ;; Surveillance Sweep
+  (testing "Basic test"
+    (do-game
+      (new-game (default-corp ["Restructured Datapool" "Surveillance Sweep" "Data Raven"])
+                (default-runner ["Scrubbed"]))
+      (is (= 0 (:tag (get-runner))) "Runner should start with no tags")
+      (play-from-hand state :corp "Surveillance Sweep")
+      (play-and-score state "Restructured Datapool")
+      (let [rd-scored (get-scored state :corp)]
+        (card-ability state :corp rd-scored 0)
+        (is (not (= :waiting (->> (get-corp) :prompt first :prompt-type))) "Surveillance Sweep on works during a run")
```

Comment:
single thread -> should be fine here

Reply:
Also, can denest this with `not=` instead of `(not (= ...))`

Could even move the `not=` check inside the thread macro.

Reply:
What would that look like?